editor.planx.uk/src/components/Feedback/index.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+import Feedback from "./index";
+
+const meta: Meta<typeof Feedback> = {
+  title: "Design System/Molecules/Feedback",
+  component: Feedback,
+};
```

Comment:
Within Storybook UI, felt a bit hard/confusing to tell the difference between `Feedback` & `FeedbackPhaseBanner` - should we only include `Feedback` as it's basically the interactive version of `FeedbackPhaseBanner` ?

Reply:
I've dropped the commit 👍